subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/suite/filter/TestsFilter.kt

```diff
@@ -16,6 +17,10 @@ interface TestsFilter {
                 name, "test has SkipSdk with value sdk=$sdk"
             )
 
+            class HaveFlakyAnnotation(name: String, sdk: Int) : Excluded(
```

Comment:
```suggestion
            class HasFlakyAnnotation(name: String, sdk: Int) : Excluded(
```
"A _test_ has flaky annotation." - is it correct?